dev_guide/projected_volumes.adoc

```diff
@@ -0,0 +1,289 @@
+* xref:downward_api.adoc#dev-guide-downward-api[Downward API]
+
+All sources are required to be in the same namespace as the pod. 
+
+////
+For more details, see the [all-in-one volume design document](https://github.com/kubernetes/community/blob/{{page.githubbranch}}/contributors/design-proposals/all-in-one-volume.md).
+////
+
+[[dev-guide-projected-volumes-example]]
+.Pod with a secret, a downward API, and a configmap
+====
+
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: volume-test
+spec:
+  containers:
+  - name: container-test
+    image: busybox
+    volumeMounts: <1>
+    - name: all-in-one
+      mountPath: "/projected-volume"<2>
+      readOnly: true <3>
+  volumes: <4>
+  - name: all-in-one <5>
+    projected:
+      sources:
+      - secret:
+          name: mysecret <6>
+          items:
+            - key: username
+              path: my-group/my-username <7>
+      - downwardAPI: <8>
+          items:
+            - path: "labels"
+              fieldRef:
+                fieldPath: metadata.labels
+            - path: "cpu_limit"
+              resourceFieldRef:
+                containerName: container-test
+                resource: limits.cpu
+      - configMap: <9>
+          name: myconfigmap
+          items:
+            - key: config
+              path: my-group/my-config
+----
+
+<1> Add a `volumeMounts` section for each container that needs the secret. 
+<2> Specify a path to an unused directory where the secret will appear.
+<3> Set `readOnly` to `true`.
+<4> Add a `volumes` block to list each projected volume source. 
+<5> Spececify any name for the volume.
+<6> Add a secret. Enter the name of the secret object. Each secret you want to use must be listed.
+<7> Specify the path to the secrets file under the `mountPath`. Here, the secrets file is in *_/projected-volume/my-group/my-config_*.
+<8> Add a Downward APUI source.
+<9> Add a ConfigMap source.
+====
+
+[NOTE]
+====
+If there are multiple containers in the pod, each container needs a `volumeMounts` section, but only one `volumes` section is needed.
+====
+
+
+.Pod with multiple secrets with a non-default permission mode set
+====
+
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: volume-test
+spec:
+  containers:
+  - name: container-test
+    image: busybox
+    volumeMounts:
+    - name: all-in-one
+      mountPath: "/projected-volume"
+      readOnly: true
+  volumes:
+  - name: all-in-one
+    projected:
+      sources:
+      - secret:
+          name: mysecret
+          items:
+            - key: username
+              path: my-group/my-username
+      - secret:
+          name: mysecret2
+          items:
+            - key: password
+              path: my-group/my-password
+              mode: 511
+----
+
+====
+
+[NOTE]
+====
+The `defaultMode` can only be specified at the projected level and not for each
+volume source. However, as illustrated above, you can explicitly set the `mode`
+for each individual projection.
```

Comment:
I used the configs in a TC as a basis for this change. Please let me know if this is what you expected. I added the defaultMode to both examples in this section.  
OCP-12427 - All-in-one volume: Permission mode work well in whole volume and individual resources.